src/Provider/Provider.Web/Controllers/Part1/DatesController.cs

```diff
@@ -39,6 +41,7 @@ public async Task<IActionResult> Dates(DatesEditModel m, [FromQuery] bool wizard
             if (!ModelState.IsValid)
             {
                 var vm = await _orchestrator.GetDatesViewModelAsync(m);
+                vm.CanShowTrainingErrorHint = response.Errors.Errors.Any(e => e.ErrorCode == ErrorCodes.TrainingExpiryDate);
```

Comment:
I'd call it "ShouldShowTrainingErrorHint". This indicates that we want it displayed, rather than it is permitted to show it in the circumstances that we wish to show it.